activity_hub.php

```diff
@@ -265,33 +265,29 @@ function summarize_stage($stage, $desired_states, $show_filtered_projects=FALSE,
 {
     global $pguser, $n_projects_in_state_, $n_projects_transitioned_to_state_;
 
-    // Get the stage identifier.
-    $stage_icon_url = get_dyn_image_url_for_file("stage_icons/{$stage->id}");
```

Comment:
side note, are code_images aren't in revision control it seems. Is that intentional?

Reply:
Actually, yes. The graphics in `code_images` are a hacky way to maintain the text-in-graphics that have been on the site for a **long** time for English users while providing fallbacks to the actual text for non-English users and other consumers of the code. [I told you it was hacky.]

I hope/expect to be changing many of these to use text as we work to support the styling work @srjfoo is working on. As you can imagine that's likely going to be a contentious change.